src/desktop/apps/artist/components/cta.js

```diff
@@ -1,8 +1,13 @@
-import $ from "jquery"
-import metaphysics from "lib/metaphysics.coffee"
-import Artist from "desktop/models/artist.coffee"
+import * as $ from "jquery"
 import { data as sd } from "sharify"
-import ArtistPageCTAView from "desktop/components/artist_page_cta/view.coffee"
+
+const metaphysics = require("lib/metaphysics.coffee")
```

Comment:
What's the reason for switching to using `require` instead of ES6 imports here?

Reply:
I had recommended this thinking that this file was `.ts` rather than `.js`, typescript requires coffee to be required -- now seeing that this file is `.js`, should probably be renamed to `.ts` as well.